zipline/pipeline/domain.py

```diff
@@ -71,6 +71,27 @@ def data_query_cutoff_for_sessions(self, sessions):
             "available" on each session.
         """
 
+    @default
+    def adjust_date(self, dt):
```

Comment:
Thoughts on renaming this to `adjust_forward` or `roll_forward`? `adjust` doesn't tell me as a reader which way the adjustment goes.

Reply:
true, changed to `roll_forward`